src/base/bittorrent/session.cpp

```diff
@@ -1511,12 +1511,12 @@ void Session::configurePeerClasses()
 
 void Session::adjustLimits(libt::session_settings &sessionSettings)
 {
-    //Internally increase the queue limits to ensure that the magnet is started
+    // Internally increase the queue limits to ensure that the magnet is started
     int maxDownloads = maxActiveDownloads();
     int maxActive = maxActiveTorrents();
 
-    sessionSettings.active_downloads = maxDownloads > -1 ? maxDownloads + m_extraLimit : maxDownloads;
-    sessionSettings.active_limit = maxActive > -1 ? maxActive + m_extraLimit : maxActive;
+    sessionSettings.active_downloads = (maxDownloads > -1) ? maxDownloads + m_extraLimit : maxDownloads;
+    sessionSettings.active_limit = (maxActive > -1) ? maxActive + m_extraLimit : maxActive;
```

Comment:
`(maxActive + m_extraLimit)`

Reply:
Done, also added the parentheses to the 2 instances of `diskCacheSize() * 64`